odoo/addons/base/models/ir_rule.py

```diff
@@ -56,6 +56,7 @@ def _eval_context(self):
             'time': time,
             'company_ids': self.env.companies.ids,
             'company_id': self.env.company.id,
+            'country_ids': (self.env.companies & self.env.user.company_ids).country_id.ids,
```

Comment:
That could be done... but not without looking at the whole big scheme of the framework, and the use case here doesn't justify it, I think

Reply:
It is not linked to the `groups` like feature.

The alternative here is to have a `account.disallowed.expenses.category.template`, and to instantiate the template for each company when installing the CoA, knowing we will have duplicated data because these are not dependant on the company but on the country.

Reply:
Or I could just make it visible for all companies; idk...